### ISSUE TITLE
Opening DevTools Console with F12 causes requests to fail when using setRequestInterception(true)

### ISSUE THREAD

**Issue by clwheeler** [puppeteer/puppeteer#3337]
### Steps to reproduce

**Tell us about your environment:**

* Puppeteer version: 1.8.0
* Platform / OS version: Mac OS High Sierra 10.13.6 (17G65)
* URLs (if applicable): Multiple (https://cloud.google.com, https://www.evo.com)
* Node.js version: v8.11.3

**What steps will reproduce the problem?**

When running in `{headless:false}` mode and using `setRequestInterception(true)`,  requests will fail if the DevTools console is opened from within Chromium using F12. The request timeout does not occur if the console is opened at launch using the `devtools:true` configuration, and does not occur if the `setInterceptionRequest(true)` is not set.

This issue does not appear to occur in earlier versions (v1.6.2).

```
(async () => {
    try {
        const browser = await puppeteer.launch({ headless: false });
        var pages = await browser.pages()
        const page = pages[0];

        await page.setRequestInterception(true);
        page.on('request', (req) => { req.continue(); });
        await page.goto('https://cloud.google.com/');
    }
    catch(e) {
        // handle uncaught promise errors
        console.log(e)
    }

})();
```

- Launch puppeteer with headless:false
- After initial page load, open devtools console with F12
- Refresh page

**What is the expected result?**
Requests complete.

**What happens instead?**
Requests fail, typically with Navigation Timeout Exceeded error.

**Comment by SimonSchick** [comment]
@clwheeler to clarify, when you log these requests do they ever fire the `requestfinished` or `requesterror` events? I'm having a similiar issue right now where requests are started but they never finish.

**Comment by nhhockeyplayer** [comment]
seeing same behavior, cant intercept posts

**Comment by aslushnikov** [comment]
@clwheeler works just fine for me with Puppeteer v1.17.0. Since v1.8.0, we changed *a lot* in request interception - so please give it another try.

> seeing same behavior, cant intercept posts

@nhhockeyplayer I'd appreciate if you file a separate issue with a code snippet that reproduces the behavior for you, together with the node.js / puppeteer / chrome versions information. Thanks!

**Comment by clwheeler** [comment]
This seems to be resolved in 1.17. Thank you!
